src/network/networkd-fdb.c

```diff
@@ -146,6 +148,19 @@ static int fdb_entry_configure(Link *link, FdbEntry *fdb_entry) {
                         return log_link_error_errno(link, r, "Could not append NDA_VLAN attribute: %m");
         }
 
+        if (fdb_entry->ifname) {
+                Link *l;
+
+                r = link_get_by_name(link->manager, fdb_entry->ifname, &l);
+                if (r < 0)
+                        log_link_warning_errno(link, r, "Failed to resolve ifname %s: %m", fdb_entry->ifname);
```

Comment:
If this fails, then skip to configure this entry. We should not configure incomplete entries.